manual/guides/pullrequests.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: manual
+title: Pull Requests
+subtitle: "What to check in a PR Review"
+permalink: /manual/guides/pullrequests
+---
+
+# Reviewing a Pull Request
+
+- [Recommendations from Cyclus PRs](https://fuelcycle.org/kernel/pr_review.html)
+
+- [Common mistakes and errors](https://blog.scottnonnenberg.com/top-ten-pull-request-review-mistakes/)
+
+Below is a checklist that lists many aspects that reviews should
+consider before approving a pull request. As a reminder, the
+[ARFC Code of Conduct](http://arfc.github.io/manual/coc) still applies
+in pull request reviews. It's important to practice proper reviewing
+while maintaining respect.
+
+```markdown
+- [ ] Read the PR description
+- [ ] Read through all the changes, considering the following questions.
+  - [ ] Is there anything you can praise about this PR? Start with
+        praise.
+  - [ ] Are variable names brief but descriptive?
+  - [ ] Are new/changed functions no longer than a paragraph?
+  - [ ] Do all function parameters have default values where appropriate?
+  - [ ] Is the code clear and clean? (see Robert C. Martin's Clean Code)
```

Comment:
add link to the Clean Code book.

Reply:
Should I use a link to a free PDF that I found or something like the Amazon page where you can buy the book?

Reply:
The UIUC library or the o'reilly access linked there:

- https://i-share-uiu.primo.exlibrisgroup.com/discovery/fulldisplay?docid=alma99944155312205899&context=L&vid=01CARLI_UIU:CARLI_UIU&tab=LibraryCatalog&lang=en
- https://learning.oreilly.com/library/view/-/9780136083238/?ar